pkg/alerts/policies.go

```diff
@@ -33,37 +33,58 @@ type Policy struct {
 	UpdatedAt          *serialization.EpochTime `json:"updated_at,omitempty"`
 }
 
-// ListPoliciesParams represents a set of filters to be
-// used when querying New Relic alert policies.
+// QueryPolicy is similar to a Policy, but the resulting GraphQL objects are string IDs in the JSON response.
+type QueryPolicy struct {
+	ID                 int                    `json:"id,string"`
+	IncidentPreference IncidentPreferenceType `json:"incidentPreference"`
+	Name               string                 `json:"name"`
+}
+
+type QueryPolicyInput struct {
+	IncidentPreference IncidentPreferenceType `json:"incidentPreference"`
+	Name               string                 `json:"name"`
+}
+
+type QueryPolicyCreateInput struct {
+	QueryPolicyInput
+}
+
+type QueryPolicyUpdateInput struct {
+	QueryPolicyInput
+}
+
+// ListPoliciesParams represents a set of filters to be used when querying New
+// Relic alert policies.
 type ListPoliciesParams struct {
 	Name string `url:"filter[name],omitempty"`
 }
 
 // ListPolicies returns a list of Alert Policies for a given account.
-func (alerts *Alerts) ListPolicies(params *ListPoliciesParams) ([]Policy, error) {
```

Comment:
Why the move away from the full name (`alerts` => `a`)?  We seem to be hit/miss on this, probably a good point to decide as a coding standard for our projects.

Reply:
Short is my preference, single letter if possible that matches the object.  Also something that doesn't overlap with the package name also seems good.

Reply:
Agreed. Seems like a pretty easy one to agree to a standardize for us. The alerts pkg seems to use `alerts *Alerts` and the entities pkg uses `e *Entities` 🤔 No strong preference to either one.

Reply:
One quick way to check for consistency is in the [godocs](https://godoc.org/github.com/newrelic/newrelic-client-go/pkg/alerts)

Reply:
The `ANSI C` in me says `a` is totally fine, the rest of my dev senses say that single letter vars are bad :mkay:  This isn't a blocker, I know we do this elsewhere, and should start a standards doc...